server/src/main/java/io/druid/server/coordination/ServerType.java

```diff
@@ -19,10 +19,30 @@
 
 package io.druid.server.coordination;
 
+/**
+ * This enum represents types of druid services that hold segments.
```

Comment:
The peons will hold segments for some time as they're indexing, before they're handed off